provisioner/registryv1/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 	}
 
 	// TODO: derive pod namespace from the pod that this process is running in.
-	ns := "kuberpak-system"
+	ns := "registryv1-system"
```

Comment:
Hmm... this reads weird? Maybe something like `rukpak-system` might make more sense in the short term while we figure out how multiple provisioners will be structured on-cluster?